build.gradle

```
@@ -24,11 +24,22 @@ repositories {
 }
 
 dependencies {
-//    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    // Spring Boot
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
+
+    // test
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+
+    // Lombok
     compileOnly 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+
+    // h2
+    runtimeOnly 'com.h2database:h2'
+
```

띄어쓰기 많은 이유가 궁금합니다.

---

추후 의존성이 많아졌을때 구분하기 쉽게 하기 위함입니다.
```suggestion
    implementation 'org.springframework.cloud:spring-cloud-starter-openfeign'
    implementation 'org.projectlombok:lombok:1.18.16'
    annotationProcessor 'org.projectlombok:lombok:1.18.16'
    testAnnotationProcessor 'org.projectlombok:lombok:1.18.16'

    implementation 'org.springframework.boot:spring-boot-starter-web'
    implementation 'org.springframework.boot:spring-boot-starter-web-services'
    implementation 'com.google.guava:guava:30.0-jre'
    implementation 'org.apache.commons:commons-lang3:3.11'
    implementation 'commons-codec:commons-codec:1.15'
    implementation group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
    implementation group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
    implementation group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.0'
    implementation group: 'org.springframework.data', name: 'spring-data-elasticsearch', version: '4.2.1'
    testImplementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
```
이런 gradle 파일 보면, 한숨부터 푹푹 나오더라구요...